tools/releaseBuild/Images/microsoft_powershell_ubuntu18.04/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Docker image file that describes an Ubuntu16.04 image with PowerShell and SnapCraft installed from Microsoft APT Repo
+FROM mcr.microsoft.com/powershell:ubuntu-18.04
+
+ARG SNAPCRAFT_CHANNEL=stable
+
+ENV SNAPCRAFT_URL=https://api.snapcraft.io/api/v1/snaps/details/snapcraft?channel=${SNAPCRAFT_CHANNEL}
+# Install curl, used later
+# install git used by snapcraft
+RUN echo SNAPCRAFT_URL: ${SNAPCRAFT_URL} && \
+  apt-get update && \
+  apt-get dist-upgrade --yes && \
+  apt-get install --yes \
+    curl \
+    sudo \
+    jq \
+    squashfs-tools \
+    git && \
+  curl -L $(curl -H 'X-Ubuntu-Series: 16' 'https://api.snapcraft.io/api/v1/snaps/details/core' | jq '.download_url' -r) --output core.snap && \
+  mkdir -p /snap/core && unsquashfs -d /snap/core/current core.snap && rm core.snap && \
+#  curl -L $(curl -H 'X-Ubuntu-Series: 16' 'https://api.snapcraft.io/api/v1/snaps/details/core18' | jq '.download_url' -r) --output core18.snap && \
```

Comment:
Should this be present here as a comment?

Reply:
This is dead code... I'll remove